warehouse/forklift/legacy.py

```diff
@@ -400,6 +400,54 @@ def _validate_classifiers(form, field):
             )
 
 
+def _validate_dynamic(form, field):
+    dynamic_fields = set(map(str.lower, field.data or []))
+
+    disallowed_dynamic_fields = []
+
+    for field_name in ("Name", "Version", "Metadata-Version"):
```

Comment:
PEP 643 only says:

> The fields `Name` and `Version` MUST NOT be marked as Dynamic.

Should `Metadata-Version` be included here? Or should the PEP be updated?